src/survival.jl

```diff
@@ -0,0 +1,38 @@
+const LC50_PARAM = Param(0.0; bounds=(0.0, 1e9))
+const HILLCOEFFICIENT_PARAM = Param(0.1; bounds=(0.0, 100))
+
+"""
+Extends CellRule for rules of survival effect
```

Comment:
Can remove the performance hint its not always true now! (the general performance got too good so chaining rules might slow it down by increasing the page size of the function)